config/locales/en.json

```
@@ -305,24 +305,40 @@
+	"consent.cancel_consent_link": "https://catalogue.servicecanada.gc.ca/content/EForms/en/ReturningtheForm/IspCpp.html",
+	"consent.i_understand_intro": "I understand that if I do not give my consent, Service Canada:",
+	"consent.i_understand_1": "will make a decision based on the available information on my file;",
+	"consent.i_understand_2": "may stop paying me the benefits if I am already receiving them; and",
+	"consent.i_understand_3": "can require that I provide the necessary information.",
 	"section2.delay_warning_1": "will make a decision based on the available information on my file;",
 	"section2.delay_warning_2": "may stop paying me the benefits if I am already receiving them; and",
 	"section2.delay_warning_3": "can require that I provide the necessary information.",
 	"section2.delay_warning_preamble": "I understand that if I do not give my consent, Service Canada:",
-	"section2.form.signature": "Signature of applicant / authorized representative",
-	"section2.form.witness_first": "First name of witness",
-	"section2.form.witness_last": "Last name(s)",
-	"section2.form.witness_middle": "Middle name",
-	"section2.form.witness_phone": "Telephone Number",
-	"section2.form.witness_signature": "Witness Signature",
-	"section2.i_consent": "<strong>I give my consent</strong> to Service Canada to obtain medical and other personal information about me from all persons and organizations listed above. I understand that this information may help determine if I qualify or continue to qualify for CPP disability benefits.",
-	"section2.i_dont_consent": "<strong>I do not give my consent</strong> to Service Canada to obtain medical and other personal information about me from all persons and organizations listed above.",
-	"section2.preamble": "The <strong>consent for Service Canada to obtain personal information</strong> must be completed and returned with this <strong>Medical Report</strong>. Your doctor or nurse practitioner can make a photocopy of this consent for their records.",
-	"section2.title": "Consent for Service Canada to obtain personal information",
-	"section2.witness_legend": "To be completed by a witness only if the applicant signs with a mark (e.g. X)",
-	"section2.witness_preamble": "I have read the contents of this section to the applicant, who appeared to fully understand them and who made their mark in my presence.",
+	"consent.yes": "<strong>Yes</strong>, I give my consent. I understand that this information may help determine if I qualify or continue to qualify for CPP disability benefits.",
+	"consent.no": "<strong>No</strong>. I do not give my consent",
+	"consent.title": "Consent for Service Canada to obtain personal information",
+	"consent.signature_draw": "Draw it",
+	"consent.signature_type": "Type it",
+	"consent.signature_option": "How would you like to give your signature?",
+	"consent.signature_draw_instructions": "Draw your signature with your mouse, touchpad, or finger.",
```

```suggestion
	"consent.signature_draw_instructions": "Draw your signature with your mouse, touchpad, or screen.",
```
This way the list is about the tech rather than technique/digit.